lib/RagdollFactory/R6RagdollBlueprint.lua

```diff
@@ -26,6 +26,13 @@ local R6RagdollBlueprint = setmetatable({
 		["Right Leg"] = { C0 = RIGHT_HIP_ATTACHMENT_CFRAME0, C1 = HIP_ATTACHMENT_CFRAME1 },
 		["Left Leg"] = { C0 = LEFT_HIP_ATTACHMENT_CFRAME0, C1 = HIP_ATTACHMENT_CFRAME1 },
 	},
+	lowDetailModeLimbs = {
+		Head = true,
+		["Right Arm"] = true,
+		["Left Arm"] = true,
+		["Right Leg"] = true,
+		["Left Leg"] = true,
+	},
```

Comment:
I didn't include a low detail mode for R6 characters because it didn't feel necessary. The fact that it's causing an error shows that i forgot to do something though. The lowDetailModeLimbs field should have been getting treated as optional. but after testing, it clearly isn't. I'm going to reject this change and do a little digging.